server/src/logic.js

```diff
@@ -291,6 +291,20 @@ export const groupHandler = {
       }),
     );
   },
+  removeUserFromGroup(_, args, ctx) {
+    const { groupId } = args.groupUpdate;
+    return getAuthenticatedUser(ctx).then(user =>
+      Group.findById(groupId).then((group) => {
+        if (!group) {
+          return Promise.reject(Error('Invalid group!'));
+        }
+        return group.removeUser(user).then((rows) => {
```

Comment:
any time you return true/false inside an if statement like this you can always shorten :)
```
return group.removeUser(user).then(rows => !!rows);
```